decktape.js

```diff
@@ -299,8 +306,18 @@ async function exportSlides(plugin, page, printer) {
     exportedSlides      : 0,
     pdfFonts            : {},
     pdfXObjects         : {},
-    totalSlides         : await plugin.slideCount(),
+    totalSlides         : await plugin.slideCount(options.enableSubsteps),
   };
+
+  if(options.enableSubsteps===false){
```

Comment:
This is specific to Impress so that should better be added to the `configure` method of the Impress plugin.

The `options` object can be passed to the Impress plugin constructor.